includes/class-sensei-admin.php

```diff
@@ -2032,7 +2032,7 @@ function() {
 	/**
 	 * Set `window.sensei.pluginUrl` to be used from javascript.
 	 *
-	 * @since  x.x.x
+	 * @since  4.5.0
```

Comment:
Interesting to see the two different placeholders being used in the same file. :P